test/zdtm.py

```diff
@@ -1645,7 +1676,11 @@ def get_freezer(desc):
         return noop_freezer()
 
     fd = desc.split(':')
-    fr = cg_freezer(path=fd[0], state=fd[1])
+
+    if os.access("/sys/fs/cgroup/user.slice/cgroup.procs", os .F_OK):
```

Comment:
Are unified cgroups always mounted just in /sys/fs/cgroup, not /sys/fs/cgroup/unified? (I mean one can do something like https://blog.christophersmart.com/2019/12/15/enabling-docker-in-fedora-31-by-reverting-to-cgroups-v1/)

Reply:
Starting with Fedora 31 there is only cgroup2 available and it is mounted at /sys/fs/cgroup. I expect that every systemd based distribution will mount cgroup2 at that location if it switches to cgroup2 only.

Reply:
@Snorch In your example, cgroupv2 will be mounted in /sys/fs/cgroup/unified, but the freezer controller will be mounted separately in /sys/fs/cgroup/freezer/. I have checked and my changes works fine in this case.